components/IndividualArticle.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {TouchableOpacity, ScrollView, View, Text, Image, StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { TouchableWithoutFeedback } from 'react-native-gesture-handler';
```

Comment:
As you are not using this one, we can remove it.

Reply:
Weird. I deleted it already. It´s added automatically for some reason.